scripts/error-codes/codes.json

```diff
@@ -143,5 +143,46 @@
   "141": "Expected onSetChildren() to fire for a container child before its parent includes it in onSetChildren().",
   "142": "Expected onBeforeMountComponent() parent and onSetChildren() to be consistent (%s has parents %s and %s).",
   "143": "React.Children.only expected to receive a single React element child.",
-  "144": "React.PropTypes type checking code is stripped in production."
-}
\ No newline at end of file
+  "144": "React.PropTypes type checking code is stripped in production.",
```

Comment:
If the test doesn't pass, can you amend the test instead? If it specifically tests for "error codes" then we can change it to trigger some other invariant whose code didn't change.